common/tlo-parsing/tl-dependency-graph.h

```diff
@@ -8,50 +8,26 @@
 #include <unordered_set>
 
 #include "common/tlo-parsing/tl-objects.h"
-#include "common/wrappers/variant.h"
 
 namespace vk {
 namespace tl {
 
-struct TLNode {
-  vk::variant<const combinator *, const type *> tl_object;
-
-  explicit TLNode(const combinator *c) :
-    tl_object(c) {}
-  explicit TLNode(const type *t) :
-    tl_object(t) {}
-
-  bool is_type() const {
-    return vk::holds_alternative<const type *>(tl_object);
-  }
-
-  bool is_combinator() const {
-    return vk::holds_alternative<const combinator *>(tl_object);
-  }
-
-  const type *get_type() const {
-    return vk::get<const type *>(tl_object);
-  }
-
-  const combinator *get_combinator() const {
-    return vk::get<const combinator *>(tl_object);
-  }
-};
-
+struct TLNode;
 struct DependencyGraphBuilder;
 
 class DependencyGraph {
 public:
-  DependencyGraph() = default;
+  DependencyGraph();
+  ~DependencyGraph();
 
   // Во время использования этого класса необходимо, чтобы *scheme не изменялось
   explicit DependencyGraph(tl_scheme *scheme);
 
-  const std::vector<TLNode> &get_nodes() const;
```

Comment:
Both functions were removed because they had used `variant` and I had to hide `TLNode` from public interface. But as soon as I've gotten rid of `variant` it seems to me that now I can roll back the functions.